### ISSUE TITLE
Any CDN?

### ISSUE THREAD

**Issue by NatsumiHoshino** [ratiw/vuetable-2#10]
I don't see any CDN version of vuetable-2 (only 1). I tried downloading the release file but it looks like I have to build it myself, and doing so results in `npm install` failing at `chromedriver@2.24.1 install: node install.js`. Using node 6.6.0 and npm 3.10.8 by the way.

**Comment by ratiw** [comment]
@NatsumiHoshino No CDN at the moment. I'm not even sure whether it is possible to compile this version into normal javascript lib as in the previous version.

I would suggest installing using NPM or [Yarn](http://yarnpkg.com/). But if you prefer to download the zip, just move all the `src\components\*.vue` into your project. The rest is for the command line tools like webpack.

**Comment by NatsumiHoshino** [comment]
Thanks for the quick reply. I just started using vue.js in some parts of a bigger project, and I am currently using vue.js from the CDN. Will that pose a problem if I include vuetable-2 from NPM?

**Comment by ratiw** [comment]
@NatsumiHoshino I don't think so. As long as vue.js is included before your compiled code, it should work. You can try and if you get stuck, just let me know. I'm not very experience with Javascript, NPM, or ES6 though, but I'll try my best.

**Comment by tahpot** [comment]
+1 for vuetable-2 from a CDN.

I tried building via npm and just hit a world of pain and just need a vuetable-2.js file otherwise I can't use this package :(

**Comment by venomir** [comment]
+1 from a CDN.
I`ve application with vuetable and vue. I need migrate on vue2 and vuetable-2, but like a @tahpot i have a same problems.
Maybe you can make CDN file like as it was with vuetable (v. 1) ?

**Comment by ratiw** [comment]
Ok, I will try to do that, but needs to do some research on using webpack to do that. If any of you have experience on that or appropriate resource, please let me know.

**Comment by syshex** [comment]
Any news on this ?  I need to upgrade from vue to vue2 and at this moment and from all the components I'm using, this is the only one I'm missing.

**Comment by WingmanImd** [comment]
@ratiw I'm also interested in this. I tried to build it yesterday but with no luck.

**Comment by clemsontiger** [comment]
would love to see this as well.

**Comment by cristijora** [comment]
I just did a PR with a change to support a js bundle for the whole library. 
It already got merged and the first js bundle can be found in this [commit](https://github.com/ratiw/vuetable-2/commit/5040e2647c91e99ad600b0870bd00b29e1452246)
If it's really urgent for some of you you can simply just use this [file](https://raw.githubusercontent.com/ratiw/vuetable-2/5040e2647c91e99ad600b0870bd00b29e1452246/dist/vuetable-2.js) directly or wait for the next release when the whole stuff will also get uploaded to cdn.

**Comment by clemsontiger** [comment]
@cristijora, thanks for this.

**Comment by clemsontiger** [comment]
@cristijora ,do you have a working example with the vuetable-2 file? I'm having some trouble with getting the paging setup. it shows the info, but not the paging.

~~~
<vuetable-pagination-info 
    ref="paginationInfo"                                 
    info-template="{from} to {to} of {total} records">
</vuetable-pagination-info>
<vuetable-pagination 
    ref="pagination" 
    @vuetable-pagination:change-page="onChangePage">
</vuetable-pagination>
~~~
it ends up displaying the pagination info twice:
~~~
1 to 20 of 200 records
Displaying 1 to 20 of 200 items
~~~

**Comment by cristijora** [comment]
Hey @clemsontiger 
Here is a [fiddle](http://jsfiddle.net/z11fe07p/1202/)
Thanks for pointing out the issue. There was a typo in my [PR](https://github.com/ratiw/vuetable-2/pull/94) where I registered the pagination info component twice.
For now you can register your component the way I did in the fiddle or:
`Vue.component('vuetable-pagination', Vuetable.VuetablePagination);`
I added a new [PR](https://github.com/ratiw/vuetable-2/pull/96) with the fix

**Comment by clemsontiger** [comment]
@cristijora thanks! one more thing, I'm trying to change a class for the pagination and it ends up throwing an error. I tried the same on the vue-table-2 tutorial and it worked, so I believe it has to do with the js file version. I tried this on your previous [fiddle](http://jsfiddle.net/z11fe07p/1202/) as well and get the same error.

TypeError: cannot read property 'first' of undefined

~~~
<vuetable-pagination ref="pagination" :css="{activeClass: 'test'}"  @vuetable-pagination:change-page="onChangePage">  </vuetable-pagination>      
~~~

Seems to give this regardless of what classes I'm setting, and only goes away if I remove the ":css="..." attribute on the vuetable-pagination.

**Comment by florisje** [comment]
@clemsontiger did you ever fix that problem? been struggling with this today...

**Comment by clemsontiger** [comment]
@florisje Unfortunately no, I believe I ended up just working with the built in classes, or went into the vuetable-2.js and modified the css prop for the various pagination classes directly. Haven't gotten back around to taking another look at it yet.

**Comment by florisje** [comment]
Your comment pointed me in the right direction. The icons css section moved to css.pagination.
This change caused that:
https://github.com/ratiw/vuetable-2/commit/1502556f77e676798d9540e9e60be222cb8d5292
(checkout the screenshot with our error)

**Comment by ratiw** [comment]
@praveenkarmanya That was actually for another library, not for Vuetable-2.
